Kconfig

```diff
@@ -11,7 +11,7 @@ config AOS_DOMD_ID
 
 config AOS_SM_VCHAN_PATH
 	string "Path to the SM vchan"
-	default "/tmp/vchan/sm"
+	default "/local/domain/1/tmp/vchan/sm"
```

Comment:
I guess you should create this path based on AOS_DOMD_ID. Or remove AOS_DOMD_ID setting at all.